docs/docs/tutorial.md

```diff
@@ -549,7 +549,7 @@ var CommentForm = React.createClass({
     var author = this.state.author.trim();
     var text = this.state.text.trim();
     if (!text || !author) {
-      return;
+      return false;
```

Comment:
Glad to here. Maybe you can open a PR that updates the example's React dependency?